Makefile

```diff
@@ -26,7 +26,7 @@ SONAME_MAJOR := 0
 SONAME_MINOR := 0
 
 # OS-specific bits
-ifeq ($(shell uname),Darwin)
+ifneq ($(shell $(CC) -print-multiarch | grep -c darwin),0)
```

Comment:
This doesn't work if `CC` is GNU GCC, which returns nothing for `-print-multiarch`.